packages/wonder-blocks-modal/src/components/modal-panel.tsx

```diff
@@ -180,20 +180,20 @@ const themedStylesFn: ThemedStylesFn<ModalDialogThemeContract> = (theme) => ({
         // programmatic focus. This is a workaround to make sure the focus
         // outline is visible when this control is focused.
         ":focus": {
-            outlineWidth: theme.border.width,
-            outlineColor: theme.border.color,
+            outlineWidth: theme.root.border.width,
+            outlineColor: theme.panel.color.border,
```

Comment:
Would it make sense for this to be `theme.root.color.border`? Or for the outlineWidth/borderRadius to be `theme.panel.border.width` and `theme.panel.border.radius` so using root or panel styles is consistent?